tests/components/camera/test_webrtc.py

```diff
@@ -52,6 +52,8 @@
 class TestProvider(CameraWebRTCProvider):
     """Test provider."""
 
+    __test__ = False  # prevent test collection of class by pytest
```

Comment:
Or just rename the class to eg `SomeTestProvider`.

Reply:
Do you have any good suggestions? `__test__` works fine. We use it somewhere else in our test suite already.

Reply:
`SomeTestProvider`

Reply:
> `SomeTestProvider`

Is that better though? If you want me too, I'll change it. Just feel like I'd still prefer `TestProvider` slightly.